src/cloudant/account.py

```diff
@@ -65,10 +62,7 @@ def __init__(self, cloudant_user, auth_token, **kwargs):
 
     def connect(self):
         """
-        _connect_
-
-        Start up an auth session for the account
-
+        Starts up an auth session for the client.
```

Comment:
Added "cookie authentication" wording to the description.  Done in b19046696b22d6b3fb89e04d7d4b7a86ca7d9c41.